.githooks/pre-push

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -euo pipefail
+
+: "${GIT_ROOT:="$(git rev-parse --show-toplevel)"}"
+BRANCH="$(git rev-parse --abbrev-ref HEAD)"
+PROTECTED="main"
+
+
+if [[ "$BRANCH" == "$PROTECTED" ]]; then
+	echo "HOOK: Pushing to protected branch ($PROTECTED) 🙊"
+	echo "HOOK: Use git push --no-verify to force this operation."
+	exit 1
```

Comment:
Guess this only protects us devops/core admins since only we can push directly on protected branches.

Reply:
Indeed, which is also protected in our github config. Belts and suspenders 🙏